### ISSUE TITLE
[RFE] Use systemd mount units instead of fstab by default

### ISSUE THREAD

**Issue by rahulsundaram** [rhinstaller/anaconda#1788]
It is easier to take advantage of systemd functionality if Anaconda didn't use fstab by default and rely on systemd-fstab-generator and instead just shipped with systemd unit files for the mount points

**Comment by jkonecny12** [comment]
Interesting idea, we will investigate that when we found some time to spare.

**Comment by keszybz** [comment]
No, please don't. `fstab` is familiar to everybody, and there's is not gain from using units in this case. Things are more verbose and more obscure, but no functionality is gained (*).

(*) OK, there are corner cases like adding special dependencies or more esoteric options that are possible with a mount unit and not with fstab, but I don't think anaconda would or should be using any of those.

**Comment by poncovka** [comment]
Thank you for reporting this issue! The issue tracker will be closed on 3 June 2019. Please make sure that your issue is also reported at https://bugzilla.redhat.com/. We are handling a high number of issues and many of them involve other components. Bugzilla allows us to better process your issues and collaborate on them with other teams. We are sorry for the inconvenience.
